images/hook/entrypoint.sh

```diff
@@ -93,6 +93,11 @@ if [ $1 = "update" ]; then
     # check for readiness of prometheus pod
     kubectl --namespace monitoring wait --for=condition=ready pod prometheus-k8s-0
 
+
+    # Remove unused nethealth objects
+    # Todo: can be removed when upgrades from gravity 7.0 are no longer supported.
+    /opt/bin/kubectl delete -n monitoring servicemonitor/nethealth || true
```

Comment:
As I wrote in the previous PR could you use `kubectl delete --ignore-not-found`?

Reply:
probably, I just don't know off hand if that's a newish flag or what versions it works in. So on a backport it doesn't seem worth it.